From 636bf6f62c4f7f9fc64d6e9c47602029b4a3b40f Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Fri, 6 Jul 2018 10:01:42 +0800
Subject: [PATCH 4082/5242] MLK-18792-2 ARM64: dts: imx8mq: remove
 unneccessary interrupt-parent property

commit  3526dc8e37a62ff18b02a78d3df55ea56cea574d from
https://source.codeaurora.org/external/imx/linux-imx.git

There is a interrupt-parent property in root node, no need
to set it again in subnode.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
(cherry picked from commit 899f4e6b4c0f2e372079db979a0a802cd6167bcd)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index e566f6f..ce9880f 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -455,7 +455,6 @@
 		reg = <0x0 0x32e2d000 0x0 0x1000>;
 		interrupts = <GIC_SPI 18 IRQ_TYPE_LEVEL_HIGH>;
 		interrupt-controller;
-		interrupt-parent = <&gpc>;
 		#interrupt-cells = <2>;
 		nxp,irqsteer_chans = <2>;
 		nxp,endian = <1>;		/* MSB */
@@ -595,7 +594,6 @@
 		interrupts = <GIC_SPI 16 IRQ_TYPE_LEVEL_HIGH>,
 					<GIC_SPI 25 IRQ_TYPE_LEVEL_HIGH>;
 		interrupt-names = "plug_in", "plug_out";
-		interrupt-parent = <&gpc>;
 		fsl,cec;
 		status = "disabled";
 
@@ -800,7 +798,6 @@
 		clocks = <&clk IMX8MQ_CLK_ECSPI1_ROOT>,
 			 <&clk IMX8MQ_CLK_ECSPI1_ROOT>;
 		clock-names = "ipg", "per";
-		interrupt-parent = <&gpc>;
 		status = "disabled";
 	};
 
@@ -848,7 +845,6 @@
 		clocks = <&clk IMX8MQ_CLK_UART1_ROOT>,
 			<&clk IMX8MQ_CLK_UART1_ROOT>;
 		clock-names = "ipg", "per";
-		interrupt-parent = <&gpc>;
 		status = "disabled";
 	};
 
@@ -954,7 +950,6 @@
 			compatible = "snps,dwc3";
 			reg = <0x0 0x38100000 0x0 0x10000>;
 			interrupts = <GIC_SPI 40 IRQ_TYPE_LEVEL_HIGH>;
-			interrupt-parent = <&gpc>;
 			phys = <&usb3_phy0 0>, <&usb3_phy0 1>;
 			phy-names = "usb2-phy", "usb3-phy";
 			power-domains = <&usb_otg1_pd>;
@@ -995,7 +990,6 @@
 			compatible = "snps,dwc3";
 			reg = <0x0 0x38200000 0x0 0x10000>;
 			interrupts = <GIC_SPI 41 IRQ_TYPE_LEVEL_HIGH>;
-			interrupt-parent = <&gpc>;
 			phys = <&usb3_phy1 0>, <&usb3_phy1 1>;
 			phy-names = "usb2-phy", "usb3-phy";
 			power-domains = <&usb_otg2_pd>;
-- 
1.7.9.5

